modules/oc-mirror-operator-catalog-filtering.adoc

```diff
@@ -121,7 +117,7 @@ mirror:
         channels
           - name: stable
 ----
-|The head bundle for the selected channel of that package.
+|The head bundle for the selected channel of that package. The `defaultChannel` should be used in case the filtered channels are not the default.
```

Comment:
```suggestion
|The head bundle for the selected channel of that package. You must use the `defaultChannel` field in case the filtered channels are not the default.
```

- Avoid using should
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s

> Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed. For example, when parameters must be reset, write “Reset the parameters” and not “You should reset the parameters”.

- Object name must be specified per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references
- Use active voice